### PR TITLE
Update version and mark new features

### PR DESCRIPTION
**Pull request by ali-ince**

### CODE REVIEW

File: antora.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+version: '5'
 start_page: ROOT:index.adoc
 nav:
 - modules/ROOT/nav.adoc
 
 asciidoc:
   attributes:
     theme: docs
-    connector-version: '5.0.3'
+    connector-version: '5'
```
Comment (lidiazuin) [inline antora.yml:11]: I believe here you can leave it annotated with the minor version, so the homepage will inform that

File: modules/ROOT/pages/configuration.adoc [hunk 2]
```diff
@@ -77,16 +77,23 @@ documentation on all possible configuration options for Neo4j Drivers, see the l
 4+|*Driver options*
 
 |`url`
-|The url of the Neo4j instance to connect to.
-You can provide a comma-separated list of valid Neo4j URIs like: `neo4j://localhost,neo4j://foo.bar,neo4j://foo.bar.baz:7783`, the first one
-will be used as base URL and the others as resolvers just as explained
-link:https://neo4j.com/docs/java-manual/current/client-applications/#java-driver-resolver-function[here, window=_blank]
+a|The url of the Neo4j instance to connect to.
+
+When provided with a comma-separated list of URIs, link:[Resolver function] feature of the driver will be activated, and the first URI will be used as original host while the rest are treated as resolver function outputs.
```
Comment (lidiazuin) [inline modules/ROOT/pages/configuration.adoc:82]:
```suggestion
When provided with a comma-separated list of URIs, the link:https://neo4j.com/docs/java-manual/current/client-applications/#java-driver-resolver-function[resolver function] feature of the driver will be activated.
The first URI will be used as original host while the rest are treated as resolver function outputs.
```

File: modules/ROOT/pages/configuration.adoc [hunk 3]
```diff
@@ -77,16 +77,23 @@ documentation on all possible configuration options for Neo4j Drivers, see the l
...
-|The authentication methods to be used: `none`, `basic`, `kerberos`, `custom`, `bearer`.
-More information link:https://neo4j.com/docs/java-manual/4.4/client-applications/#java-driver-authentication[here, window=_blank]
+a|
+The authentication methods to be used:
+
+* `none`
+* `basic`
+* `kerberos`
+* `custom`
+* `bearer` label:new[Introduced in 5.1]
```
Comment (lidiazuin) [inline modules/ROOT/pages/configuration.adoc:94]: Do you think if we make it shorter like this, it's still understandable? It just looks better on the website

```suggestion
* `bearer` label:new[v.5.1]
```

File: modules/ROOT/pages/playground.adoc [hunk 4]
```diff
@@ -8,9 +8,9 @@ image::colab-to-sandbox.png[Colab/PySpark to Neo4j sandbox architecture, align="
 
 These notebooks contain a set of examples that explain how the Neo4j Spark connector can fit into your data-driven workflow, and mostly important they allow you to test your knowledge with a set of exercises after each section.
 
-If you have any problem feel free to write a post in the [Neo4j community forum](https://community.neo4j.com/) or in [Discord](https://discord.com/invite/neo4j).
+If you have any problem feel free to write a post in the +If you have any problem feel free to write a post in https://community.neo4j.com[Neo4j community forum] or in https://discord.com/invite/neo4j[Discord].
```
Comment (lidiazuin) [inline modules/ROOT/pages/playground.adoc:11]:
```suggestion
If you have any problem feel free to write a post in the https://community.neo4j.com[Neo4j community forum] or in https://discord.com/invite/neo4j[Discord].
```

File: modules/ROOT/pages/reading.adoc [hunk 5]
```diff
@@ -83,7 +83,7 @@ When `string` is set, it coerces all the properties to String, otherwise it trie
 |`true`
 |No
 
-|`pushdown.limit.enabled`
+|`pushdown.limit.enabled` label:new[Introduced in 5.1]
```
Comment (lidiazuin) [inline modules/ROOT/pages/reading.adoc:86]: Same as the other previous suggestion

```suggestion
|`pushdown.limit.enabled` label:new[v.5.1]
```

**Review by lidiazuin — CHANGES_REQUESTED** [review]
Thanks Ali, I have left a few suggestions

File: modules/ROOT/pages/gds.adoc [hunk 6]
```diff
@@ -18,7 +18,7 @@ All of the link:{url-neo4j-gds-manual}[functionality of GDS^] is used by issuing
 accessible via Spark, because the Neo4j Connector for Apache Spark can issue Cypher queries and read their results back.  This combination means
 that you can use Neo4j and GDS as a graph co-processor in an existing ML workflow that you may implement in Apache Spark.
 
-== GDS first-class support
+== GDS first-class support label:new[Introduced in 5.1]
```
Comment (ali-ince) [inline modules/ROOT/pages/gds.adoc:21]: @lidiazuin should we also update these to the shorter version, wdyt?

File: antora.yml (hunk 1 above)
Comment (lidiazuin) [reply antora.yml:11]: So this is actually a shortcut to add the version of the connector automatically to the index page, as you can check here https://github.com/neo4j/docs-spark/blob/dev/modules/ROOT/pages/index.adoc?plain=1 
If you want to be specific about the minor version, then that's the place

File: modules/ROOT/pages/gds.adoc (hunk 6 above)
Comment (lidiazuin) [reply modules/ROOT/pages/gds.adoc:21]: We could, but in this case it is the title of a section so the size of the label is not so disruptive

**Review by lidiazuin — APPROVED** [review]

**Pull request merged**